openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -689,7 +689,7 @@ For cryptographic binding, the Client has the following options to provide crypt
 A Client makes a Credential Request to the Credential Endpoint by sending the following parameters in the entity-body of an HTTP POST request using the `application/json` media type.
 
 * `format`: REQUIRED. Format of the Credential to be issued. This Credential format identifier determines further parameters required to determine the type and (optionally) the content of the credential to be issued. Credential Format Profiles consisting of the Credential format specific set of parameters are defined in (#format_profiles).
-* `proof`: OPTIONAL. JSON object containing proof of possession of the key material the issued Credential shall be bound to.  The `proof` object MUST contain a following claim:
+* `proof`: OPTIONAL. JSON object containing proof of possession of the key material the issued Credential shall be bound to.  The `proof` object is REQUIRED if the `proof_of_possession_required` parameter in the issuer metadata for the submitted credential format is `true`. The `proof` object MUST contain a following claim:
```

Comment:
```suggestion
* `proof`: OPTIONAL. JSON object containing proof of possession of the key material the issued Credential SHALL be bound to.  The `proof` object is REQUIRED if the `proof_of_possession_required` parameter in the issuer metadata for the submitted Credential format is `true`. The `proof` object MUST contain a following claim:
```

Reply:
Jacob, please pull the most recent version of the spec. the text has been updated to `proof: OPTIONAL. Object containing the proof of possession of the cryptographic key material the issued Credential would be bound to.`

Giuseppe, this specification uses MUST, not shall for normative statements.

Reply:
Since we decided in #97 to drop the "JSON" in most places, please remove it here as well.

Reply:
Done.